social_core/backends/fedora.py

```diff
@@ -4,9 +4,35 @@
 """
 
 from .open_id import OpenIdAuth
+from .open_id_connect import OpenIdConnectAuth
+
+
+class FedoraOpenIdConnect(OpenIdConnectAuth):
+    """
+    Fedora OpenID Connect backend.
+
+    To use it, you need to set the SOCIAL_AUTH_FEDORA_KEY and
+    SOCIAL_AUTH_FEDORA_SECRET configuration variables to the client_id and
+    client_secret values that the Fedora Infrastructure gave you.
+    """
+
+    name = "fedora"
```

Comment:
The backend name should be different, for example:

```suggestion
    name = "fedora-oidc"
```

Reply:
Or is this really expected to be a drop-in replacement including same user identifiers?

Reply:
What I meant is user associations. For example, with OpenID I have:

![image](https://github.com/user-attachments/assets/06298240-52b6-4eaf-b633-091cfdeda058)

Will be the UID provided by OIDC same as with Open ID? If not, then the backend should get a different name so that it doesn't try to match OpenID UIDs against OIDC ones.

Reply:
Ah I see, no UIDs will be taken from the `sub` attribute according to the OpenID Connect protocol. I'll change the name, thanks for pointing this out.